brjs-sdk/sdk/libs/javascript/br/src/br/ServiceRegistryClass.js

```diff
@@ -129,6 +129,29 @@ ServiceRegistryClass.prototype.legacyClear = function() {
 	this.registry = {};
 };
 
+ServiceRegistryClass.prototype.dispose = function() {
```

Comment:
Is there a reason we are logging to the console instead of _fell_? If we should be using _fell_ than we should probably take the opportunity to log at more appropriate log levels as some of the information level log lines seem like they should really be debug level.